### PR TITLE
DOCSP-41781-Release-Notes-2-2-15

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- This ticket is for `2.2.13` but that release had no changes, see [2.2.13](https://github.com/mongodb-js/mongosh/releases/tag/v2.2.13) and [2.2.14](https://github.com/mongodb-js/mongosh/releases/tag/v2.2.14).
- Repurpose this ticket for the [2.2.15 ](https://github.com/mongodb-js/mongosh/releases/tag/v2.2.15)release notes instead.
- [See discussion in slack](https://mongodb.slack.com/archives/CUHC9R8J0/p1722338217968069).

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/mongodb-shell/DOCSP-41781/changelog/#v2.2.15

## JIRA

https://jira.mongodb.org/browse/DOCSP-41781

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66aa5d85824f175afd6f075a

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,26 @@ Release Notes
...
+*Released July 30, 2024*
+
+
+New features in this release:
+
+- :issue:`MONGOSH-1848` Added helper for ``shardMoveCollection`` and ``unshardCollection`` commands. 
+  
+Bug fixes in this release:
+
+- ``Sh.status()`` now shows a full list of tag ranges in verbose mode only :issue:`MONGOSH-1327`.
```
Comment (mabaasit) [inline source/changelog.txt:27]:
```suggestion
- ``sh.status()`` now shows a full list of tag ranges in verbose mode only :issue:`MONGOSH-1327`.
```

**Review by mabaasit — APPROVED** [review]

**Review by ltran-mdb2 — APPROVED** [review]
LGTM! Thanks @ianf-mongodb !

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,26 @@ Release Notes
...
+
+*Released July 30, 2024*
+
+
+New features in this release:
+
+- Added helper for ``shardMoveCollection`` and ``unshardCollection`` commands :issue:`MONGOSH-1848`. 
+  
+Bug fixes in this release:
+
```
Comment (sarah-olson-mongodb) [inline source/changelog.txt:26]: Before we merge, can you flip the Jira and the description, and turn them into an ordered list? 
- MONGOSH-####: Description
- MONGOSH-####: DEscription

Comment (ianf-mongodb) [reply source/changelog.txt:26]: Sure, per some of the previous RN's I used a format with `-`. 

For example v2.2.9:

```
- :issue:`MONGOSH-1785` - ``monogosh`` now supports Ubuntu 24.04. 
```

**Review by sarah-olson-mongodb — APPROVED** [review]

**Pull request merged**